shared/js/background/before-request.js

```diff
@@ -1,6 +1,6 @@
 import browser from 'webextension-polyfill'
 import EventEmitter2 from 'eventemitter2'
-const tldts = require('tldts')
```

Comment:
These changes don't seem to generally change require to import, so I'm wondering if there's a reason to change this one? If not, maybe save this for a follow-up where we adjust all the requires?

Reply:
Thanks for pointing that out - there were some changes for `esbuild` which are not needed now I decided to leave that out of the scope of this PR.